core/vm/gas_table.go

```diff
@@ -356,6 +378,28 @@ func gasCreate2Eip3860(_ *EVM, contract *Contract, stack *Stack, mem *Memory, me
 	return gas, nil
 }
 
+func gasCreate2EIP7907(evm *EVM, contract *Contract, stack *Stack, mem *Memory, memorySize uint64) (uint64, error) {
```

Comment:
Ditto, it'd be nice to extract a common function for `gasCreate2Eip3860` & `gasCreate2EIP7907`.